src/eduid_idp/shared_session.py

```diff
@@ -91,6 +95,7 @@ def common(self) -> Optional[Common]:
     def common(self, value: Optional[Common]):
         if not self._common:
             self._common = value
+        self['flag'] = f'dirty session to force saving to redis {random()}'
```

Comment:
cherrypy only saves sessions when there are changes in the session data, and the data in the namespaces only reaches cherrypy.session when saving, so we add this to trigger session saving. cherrypy.session does not have a dirty flag.